tests/tests/swfs/avm2/static_length/test.as

```diff
@@ -0,0 +1,34 @@
+
+package
+{
+    import flash.display.Sprite;
+
+    public class test extends Sprite {}
+}
+
+
+
+trace("RegExp.length is: " + RegExp.length);
+trace("String.length is: " + String.length);
+trace("XMLList.length is: " + XMLList.length);
+trace("Namespace.length is: " + Namespace.length);
+trace("ReferenceError.length is: " + ReferenceError.length);
+trace("DefinitionError.length is: " + DefinitionError.length);
+trace("ArgumentError.length is: " + ArgumentError.length);
+trace("SyntaxError.length is: " + SyntaxError.length);
+trace("VerifyError.length is: " + VerifyError.length);
+trace("SecurityError.length is: " + SecurityError.length);
+trace("EvalError.length is: " + EvalError.length);
+trace("Number.length is: " + Number.length);
+trace("RangeError.length is: " + RangeError.length);
+trace("Boolean.length is: " + Boolean.length);
+trace("XML.length is: " + XML.length);
+trace("Function.length is: " + Function.length);
+trace("TypeError.length is: " + TypeError.length);
+trace("URIError.length is: " + URIError.length);
+trace("Array.length is: " + Array.length);
+trace("uint.length is: " + uint.length);
+trace("Date.length is: " + Date.length);
+trace("Error.length is: " + Error.length);
+//trace("UninitializedError.length is: " + UninitializedError.length);
```

Comment:
why is this commented out?

Reply:
Because `UninitializedError` has not yet been implemented - should I add a comment that explains it to the file? Like:

```
// UninitializedError has not yet been implemented at the time of writing this test.
//trace("UninitializedError.length is: " + UninitializedError.length);
```

?

Reply:
Maybe:

```
// Once the class UninitializedError has been implemented, please:
// - Implement UninitializedError.length .
// - Replace the file "output.txt" with "output_original.txt".
// - Comment in the line below.
//trace("UninitializedError.length is: " + UninitializedError.length);
```

?

Reply:
Alternatively, I could look at implementing the class `UninitializedError` first (and also add `static length` for it and comment-in this line), but I do not know how involved and complex implementing that class would be, nor how to test it.

Reply:
I added some comments to the ActionScript test file.